.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+compiler: gcc
+branches:
+  only:
+    - master
+    - develop
+    - /^hotfix\/.*/
+    - /^release\/.*/
+before_script:
+  - export DISPLAY=:99.0
+  - sh -e /etc/init.d/xvfb start
+    #- xvfb-run -a   # initialize virtual framebuffer x-server for GUI testing
```

Comment:
okay, add it before "sh -e /etc/init.d/xvfb start" line (without "xvfb-run -a" part)